pkg/etcd/etcd_int_test.go

```diff
@@ -12,13 +12,21 @@ import (
 )
 
 var server *testutil.K3sServer
+var serverArgs = []string{"--cluster-init"}
 var _ = BeforeSuite(func() {
-	var err error
-	server, err = testutil.K3sStartServer("--cluster-init")
-	Expect(err).ToNot(HaveOccurred())
+	if !testutil.IsExistingServer() {
+		var err error
+		server, err = testutil.K3sStartServer(serverArgs...)
+		Expect(err).ToNot(HaveOccurred())
+	}
 })
 
 var _ = Describe("etcd snapshots", func() {
+	BeforeEach(func() {
```

Comment:
Note: This is unfortunately necessary, as calling Skip in the BeforeSuite function causes the following tests to fail, not skip. There is currently no way in Ginkgo to Skip entire test suites.